source/plain.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+function plain (array){
+	return array.reduce((accumulator, currentValue)=>{
```

Comment:
```suggestion
	return array.reduce((accumulator, currentValue) => {
```

В приличном обществе вокруг стрелки принято ставить пробелы)